api_formatter/templates/rest_framework/api.html

```diff
@@ -1,292 +1,415 @@
-{% extends "rest_framework/base.html" %}
+{% extends "rest_framework/rac_base.html" %}
 
 {% load static %}
 {% load i18n %}
 {% load rest_framework %}
 
 {% block style %}
-<link rel="stylesheet" type="text/css" href="{% static "bootstrap/css/bootstrap.min.css" %}">
+<!-- <link rel="stylesheet" type="text/css" href="{% static "bootstrap/css/bootstrap.min.css" %}"> -->
+<link rel="icon" type="image/x-icon" href="https://assets.rockarch.org/assets/img/favicon.ico">
+<link rel="stylesheet" type="text/css" href="https://assets.rockarch.org/v0.8.1/main.min.css">
 <link rel="stylesheet" href="{% static "css/styles.css" %}">
 {% endblock %}
 
-{% block title %}{% if name %}{{ name }} – {% endif %}RAC API{% endblock %}
+{% block title %}{% if name %}{{ name }} – {% endif %}api.rockarch.org{% endblock %}
 
 {% block navbar %}
-<header class="main-header">
-  <nav class="navbar navbar-expand-sm navbar-dark bg-dark mb-5" role="navigation" aria-label="{% trans "navbar" %}">
-    <div class="container">
-      <a class="navbar-brand" href="/">RAC API</a>
+<header class="header header--blue">
+  <div class="wrapper">
+    <div class="container flex">
+      <div class="header__brand header__brand--text">
+        <a href="/" class="header__brand-title">api.rockarch.org</a>
+        <p class="header__brand-subtitle">
+          The Collections API of the Rockefeller Archive Center
+        </p>
+    </div>
 
-      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-        <span class="navbar-toggler-icon"></span>
-      </button>
+    <nav class="nav-right" aria-label="Main">
+      <ul class="nav__list show-on-lg-up">
+        <li class="nav__item btn--navy">
+          <a class="nav__link" href="{% url 'agent-list' %}" id="agents">
+            Agents
+          </a>
+        </li>
+        <li class="nav__item btn--navy">
+          <a class="nav__link" href="{% url 'collection-list' %}" id="collections">
+            Collections
+          </a>
+        </li>
+        <li class="nav__item btn--navy">
+          <a class="nav__link" href="{% url 'object-list' %}" id="objects">
+            Objects
+          </a>
+        </li>
+        <li class="nav__item btn--navy">
+          <a class="nav__link" href="https://docs.rockarch.org/argo-docs" id="docs">
+            Documentation
+            <span class="material-icon" aria-hidden="true">launch</span>
```

Comment:
The launch icon is there to indicate that this link goes to another site? I'd like to hear more about the thinking for this. In general, I think if we use it here, we should make it part of our style guide and use consistently across sites (and add an aria-label). BUT, GOV UK [did away with this practice](https://designnotes.blog.gov.uk/2016/11/28/removing-the-external-link-icon-from-gov-uk/) a few years back and I've seen some critiques of its necessity.

Reply:
Yeah I was trying to indicate that you'd end up on a different site if you clicked this link. It wasn't a well-considered move, so I'm happy to remove it.